intellij/src/saros/core/ui/eventhandler/NegotiationHandler.java

```diff
@@ -78,30 +78,28 @@ public void handleIncomingProjectNegotiation(AbstractIncomingProjectNegotiation
   private void showIncomingInvitationUI(
       Project project, final IncomingSessionNegotiation negotiation) {
 
-    ApplicationManager.getApplication()
-        .invokeLater(
-            () -> {
-              JoinSessionWizard wizard =
-                  new JoinSessionWizard(project, getWindow(project), negotiation);
-              wizard.setModal(true);
-              wizard.open();
-            },
-            ModalityState.defaultModalityState());
+    EDTExecutor.invokeLater(
+        () -> {
+          JoinSessionWizard wizard =
+              new JoinSessionWizard(project, getWindow(project), negotiation);
+          wizard.setModal(true);
+          wizard.open();
+        },
+        ModalityState.defaultModalityState());
```

Comment:
Why do you use the default modality state (for multiple calls of EDTExecutor methods in this PR) instead of using your method with the signature `invokeLater(@NotNull Runnable)` that uses the default state?

Reply:
The methods implicitly using the default modality state were meant more as convenience methods. As the default modality state is most often used, I thought it would be nice to have a method were we wouldn't have to specify it.

But I didn't feel like it was necessary to replace all prior usages explicitly stating to use the default modality state with this new call.